subsys/bluetooth/audio/has_internal.h

```diff
@@ -46,6 +46,9 @@
 
 #define BT_HAS_IS_LAST 0x01
 
+/* ATT notification/indication header length (opcode + handle) */
+#define BT_HAS_ATT_NTF_HDR_LEN   3
```

Comment:
I'm not sure what I think about this. You're basically defining the length of ATT_HANDLE_VALUE_NTF/ATT_HANDLE_VALUE_IND here, right?
I think that such a definition should reside in GATT probably. Since you are only using it once, consider doing it as has been done in other places, i.e. mark with a comment as such:
`/* PDU structure is [Opcode (1)] [Handle (2)] [...] */`

Reply:
Yes, the structures are defined in `att_internal.h` (which I do not want to include).
I have a feeling that if I would did like you suggested then I would get a comment to not use magic numbers.
I'm OK with having a comment

Reply:
done

Reply:
This is a common issue for services: In order to properly truncate notifications, they need the notification header size. 

We should probably add that to `att.h`

Reply:
> Yes, the structures are defined in `att_internal.h` (which I do not want to include). I have a feeling that if I would did like you suggested then I would get a comment to not use magic numbers. I'm OK with having a comment

I'm also not a fan of magic numbers :D But seeing how widespread the use of them are in Zephyr, I think that the solution you did now is better regardless :)

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/61980#discussion_r1366690682